### PR TITLE
Cron CI improvements

### PR DESCRIPTION
**Pull request by kgerheiser**
Add options to UFS regression tests. Their tests have been updated and the `-f` option was removed.

Use `RT_COMPILER` (intel/gnu) variable to select which UFS modules are edited to allow for building with GNU.

Remove `git -C` because it doesn't work on all versions of git.

Remove `-l` from scripts.

Group logs by day.

I've had this testing on Hera over the weekend.

### CODE REVIEW

File: cron-ci/setup-cron.sh [hunk 1]
```diff
@@ -19,23 +19,27 @@ export HPC_STACK_FILE=config/stack_noaa.yaml
 
 # set machine name (hera, jet, orion, etc) so script edits correct ufs modulefiles
 export HPC_MACHINE_ID=
-export LOG_PATH=${HPC_HOMEDIR}/logs
+
+today=$(date +'%m-%d-%Y')
```
Comment (aerorahul) [inline cron-ci/setup-cron.sh]: can you make this`%Y-%m-%d`?
Will make sorting by year, month, day easier to look at.

File: cron-ci/setup-cron.sh [hunk 2]
```diff
@@ -19,23 +19,27 @@ export HPC_STACK_FILE=config/stack_noaa.yaml
...
 mkdir -p ${HPC_DOWNLOAD_PATH}
 mkdir -p ${HPC_HOMEDIR}
-mkdir -p ${LOG_PATH}
+mkdir -p ${HPC_LOG_PATH}
 
 cd $HPC_DOWNLOAD_PATH
 rm -rf hpc-stack
 
 # mm-dd-yyy-hh:mm
 hpc_logdate=$(date +'%m-%d-%Y-%R')
```
Comment (aerorahul) [inline cron-ci/setup-cron.sh]: This should also be year-month-day.

File: cron-ci/test-applications.sh [hunk 3]
```diff
@@ -1,28 +1,39 @@
...
 
 cron_dir=${HPC_DOWNLOAD_PATH}/hpc-stack/cron-ci
 
 if [[ "$TEST_UFS" == true ]]; then
+
+    cd ${HPC_DOWNLOAD_PATH}
+    rm -rf ufs-weather-model
+
     # mm-dd-yyy-hh:mm
     ufs_logdate=$(date +'%m-%d-%Y-%R')
```
Comment (aerorahul) [inline cron-ci/test-applications.sh]: here as well.

**Review by aerorahul — COMMENTED** [review]
Scripts look good.
I have one comment about the date string.
I think it should be `year-month-day`.  It makes sorting easier and follows how we usually do this sort of thing.

File: cron-ci/setup-cron.sh (hunk 1 above)
Comment (kgerheiser) [reply cron-ci/setup-cron.sh]: Ok

File: cron-ci/setup-cron.sh [hunk 4]
```diff
@@ -37,7 +37,7 @@ cd $HPC_DOWNLOAD_PATH
 rm -rf hpc-stack
 
 # mm-dd-yyy-hh:mm
```
Comment (aerorahul) [inline cron-ci/setup-cron.sh]: comment is incorrect.

File: cron-ci/test-applications.sh [hunk 5]
```diff
@@ -9,7 +9,7 @@ if [[ "$TEST_UFS" == true ]]; then
     rm -rf ufs-weather-model
 
     # mm-dd-yyy-hh:mm
```
Comment (aerorahul) [inline cron-ci/test-applications.sh]: same comment here.

**Review by aerorahul — APPROVED** [review]
approving. please look at the comments too.

**Review by aerorahul — APPROVED** [review]

**Pull request merged**